pkg/sql/catalog/catconstants/namespace.go

```diff
@@ -18,4 +18,12 @@ const (
 	// NamespaceTablePrimaryIndexID is the id of the primary index of the
 	// namespace table.
 	NamespaceTablePrimaryIndexID = 1
+
+	// NamespaceTableName is the name of the namespace table.
+	NamespaceTableName = "namespace"
```

Comment:
Perhaps reference this constant in the definition of `systemschema.NamespaceTable`?

Reply:
I do?